File: src/Chapter-2-C99/Exercises/Question2.c
//Write a program that computes the volume of a sphere with
//a 10-meter radius.

#include <stdio.h>
//Declare constant PI
#define PI 3.14f

int main(void)
{
    float r = 10.0f;
    float volume = (4.0f/3.0f)*PI*(r*r*r);
    printf("Volume is %f", volume);
    return 0;
}

File: src/Chapter-2-Rust/Exercises/Question2.rs
//Write a program that computes the volume of a sphere with
//a 10-meter radius.
use std::process::Command;
const PI: f32 = 3.14159265359;

fn main()
{
	let r = 10.0;
	let volume = (4.0/3.0)*PI*(r*r*r);
	println!("Volume is {}", volume);
	let _ = Command::new("cmd.exe").arg("/c").arg("pause").status();
}
